Modules/mathmodule.c

```diff
@@ -2016,6 +2016,69 @@ math_factorial(PyObject *module, PyObject *arg)
 }
 
 
+/*[clinic input]
+math.lcm
+
+	x as a: object
+	y as b: object
+    /
+least common multiple of x and y
+[clinic start generated code]*/
+
+static PyObject*
+math_lcm(PyObject *module, PyObject *a, PyObject *b)
+{
+    PyObject *g, *m, *f, *ab;
+	
+    a = PyNumber_Index(a);
+    if (a == NULL) {
+        return NULL;
+    }
+    b = PyNumber_Index(b);
+    if (b == NULL) {
+        Py_DECREF(a);
+        return NULL;
+    }
+    g = _PyLong_GCD(a, b);
+    if (g == NULL) {
+        Py_DECREF(g);
```

Comment:
If `g` is `NULL`, decrefing it is an error.  Don't you want to decref both `a` and `b` instead in this case?